### PR TITLE
Feature: Upload user photo

### PR DESCRIPTION
**Pull request by proxiex**
- allows a user to upload a profile image

Screenshot 2019-06-18 at 7 51 49 AM

### CODE REVIEW

File: traveler/urls.py [hunk 1]
```diff
@@ -37,3 +39,6 @@
     path('api/v1/', include('flights.urls')),
     path('api-docs/', schema_view)
 ]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (tonymontaro) [inline traveler/urls.py:44]: This runs when in development mode. What would happen in production?

File: users/views.py [hunk 2]
```diff
@@ -103,6 +107,12 @@ def put(self, request):
         if dob:
             profile_.update(dob=dob)
 
+        if profile_img:
+            user = CustomUser.objects.get(id=request.user.id)
+            file = ProfileSerializer(user, data={'profile_img': profile_img}, partial=True)
```
Comment (tonymontaro) [inline users/views.py:112]: I did observe where the uploaded image is associated with the user. Please attach a screenshot demonstrating the profile-image as part of the User.

**Review by tonymontaro — CHANGES_REQUESTED** [review]
Good job. Deploy the app before proceeding. The upload of an image looks complete but I see no relation to the user, add a screenshot of the result.

File: users/views.py (hunk 2 above)
Comment (proxiex) [reply users/views.py:112]: The screenshot has been uploaded

File: traveler/urls.py (hunk 1 above)
Comment (proxiex) [reply traveler/urls.py:44]: It will still access the same URL, just that in Debug mode it has to be specified.

**Review by tonymontaro — APPROVED** [review]
Great. I noticed that you force-pushed, I strongly discourage such practice. You should create a separate commit instead.

**Pull request merged**